docs/concepts/Security-Best-Practices.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Best Practices for a Secure Supply Chain
+description: Best practices for securing your supply chain software using NuGet & GitHub.
+author: JonDouglas
+ms.author: jodou
+ms.date: 02/08/2021
+ms.topic: conceptual
+---
+
+# Best Practices for a Secure Supply Chain
```

Comment:
```suggestion
# Best practices for a secure supply chain
```

Reply:
Looks like NuGet was converted to sentence caps for heading a while ago; that is the official standard these days.